examples/sycl/05_pvc_gemm_with_epilogues/05_pvc_gemm_with_per_col_bias.cpp

```diff
@@ -29,10 +29,13 @@
  *
  **************************************************************************************************/
 /*! \file
-    \brief CUTLASS Intel PVC Gemm with per-col-bias epilogue
+    \brief CUTLASS Intel PVC Gemm with per-col-bias epilogue (aka Linear layer)
 
-    This example constructs and executes a standard GEMM fused with a per-col-bias epilogue.
-    Aside from the epilogue operation, it is identical to 00_pvc_gemm.
+    This example constructs and executes a standard GEMM (with ColumnMajor B) fused with a
+    per-col-bias epilogue. Batch size is also treated differently in this example: when a batch size
+    is defined by e.g. `--l=16`, this defines the number of A matrices. A single B (weights) matrix
+    is used for the whole batch. Note the use of `LayoutA` but `StrideB`, as we need to communicate
```

Comment:
Could this be also made part of example name? Such as `05_pvc_gemm_single_b_with_per_col_bias.cpp`

Reply:
good idea, done :+1: